src/bilder/images/edxapp/packer.pkr.hcl

```diff
@@ -0,0 +1,95 @@
+locals {
+  timestamp = regex_replace(timestamp(), "[- TZ:]", "")
+  business_unit = "operations"
```

Comment:
Why not `mitx-online`?

Reply:
Until it gets deployed the image could be used for any of our deployments. I think it might make sense to keep it as is, at least until we get to a point where we're building explicitly for a given app. I might make that pull from a var that we can override.